addons/l10n_mx/models/template_mx.py

```diff
@@ -39,6 +39,7 @@ def _get_mx_res_company(self):
                 'tax_cash_basis_journal_id': 'cbmx',
                 'account_sale_tax_id': 'tax12',
                 'account_purchase_tax_id': 'tax14',
+                'account_cash_basis_base_account_id': 'cuenta801_01_99',
```

Comment:
You can remove `property_cash_basis_base_account_id` from the MX template. This is a mistake made by the script when the templates engine has been refactored. Otherwise, it's ready :)